pkg/velerorunner/backup.go

```diff
@@ -32,10 +32,10 @@ import (
 var log = logf.Log.WithName("controller")
 
 // RunBackup runs a Velero Backup if it hasn't been run already
-func RunBackup(c client.Client, backupNsName types.NamespacedName, assets *migapi.MigAssetCollection, logPrefix string) (*velerov1.Backup, error) {
+func RunBackup(c client.Client, backupNsName types.NamespacedName, assets *migapi.MigAssetCollection, logPrefix string, stageBackup bool) (*velerov1.Backup, error) {
 
 	vBackupExisting := &velerov1.Backup{}
-	vBackupNew := buildVeleroBackup(backupNsName.Namespace, backupNsName.Name, assets.Spec.Namespaces)
+	vBackupNew := buildVeleroBackup(backupNsName.Namespace, backupNsName.Name, assets.Spec.Namespaces, stageBackup)
```

Comment:
I wonder if it would make sense for us to pass an entire Velero Backup into `RunBackup` instead of adding this `stageBackup` bool. If we went this route, we could drop `backupNsName`, `assets`, and `stageBackup` from the param list.

Reply:
This would also make RunBackup more general in a way. You'd hand it an object definition of a Velero Backup you want to run, and it goes and does it for you.

Reply:
Maybe this would require making buildVeleroBackup public.

Reply:
+1 yeah I like that approach. I will refactor so that `buildVeleroBackup` is public and we just pass the backup object to `RunBackup`